doc/connectivity/bluetooth/api/mesh/sar_cfg.rst

```diff
@@ -43,34 +43,6 @@ Keep this in mind when defining the size of the buffers.
 
 SAR does not impose extra overhead on the access layer payload per segment.
 
-Intervals, timers and retransmission counters
-*********************************************
-
-The current stable stack implementation allows you to configure the following SAR behavior.
-
-When sending a segmented message to a unicast address, the unacknowledged segments are repeated
-the :kconfig:option:`CONFIG_BT_MESH_TX_SEG_RETRANS_COUNT` number of times before the transmission
-is considered as failed. The same option configures a number of retransmissions to a group or
-virtual address, but the transmission always succeedes after retransmitting all segments the
-configured number of times.
-
-The timeout between each retransmission to a unicast address is configured by the Kconfig option
-:kconfig:option:`CONFIG_BT_MESH_TX_SEG_RETRANS_TIMEOUT_UNICAST`. The timeout between each
-retransmission to a group or a virtual address is configured by the Kconfig option
-:kconfig:option:`CONFIG_BT_MESH_TX_SEG_RETRANS_TIMEOUT_GROUP`.
-
-The time before sending a Segment Acknowledgment message is controlled by the Kconfig options
-:kconfig:option:`CONFIG_BT_MESH_SEG_ACK_BASE_TIMEOUT`,
-:kconfig:option:`CONFIG_BT_MESH_SEG_ACK_PER_HOP_TIMEOUT` and
-:kconfig:option:`CONFIG_BT_MESH_SEG_ACK_PER_SEGMENT_TIMEOUT`, and is defined as:
-
-.. math::
-   \begin{aligned}
-   \max(&\mathtt{CONFIG\_BT\_MESH\_SEG\_ACK\_BASE\_TIMEOUT} \\
-   &+ \text{TTL} \times \mathtt{CONFIG\_BT\_MESH\_SEG\_ACK\_PER\_HOP\_TIMEOUT} \\
-   &+ \text{number of un-acked segments} \times \mathtt{CONFIG\_BT\_MESH\_SEG\_ACK\_PER\_SEGMENT\_TIMEOUT} , 400)
-   \end{aligned}
-
 Segmentation and reassembly (SAR) Configuration models
 ======================================================
```

Comment:
This heading and all the subsequent ones can now be moved one level up.
Thinking also this section can be named Intervals, timers and retransmission counters rather than SAR Config models... what do you think?